src/feed/views/feed_view.py

```diff
@@ -109,14 +113,19 @@ def get_queryset(self):
                     hubs__id__in=followed_hub_ids,
                 )
 
-        if feed_view == "popular":
-            # Only show paper and post
-            queryset = queryset.filter(
-                content_type__in=[
-                    self._paper_content_type,
-                    self._post_content_type,
-                ]
-            )
+        # Handle both popular view and following view with hot_score sorting
+        if feed_view == "popular" or (
+            feed_view == "following" and sort_by == "hot_score"
+        ):
+            # Only show paper and post (for popular view only)
+            if feed_view == "popular":
```

Comment:
So if `feed_view` is "following" we show other content types too? What would these be?